editoast/src/views/timetable/occupancy_blocks.rs

```diff
@@ -66,20 +64,30 @@ pub(super) async fn compute_batch_signal_updates<'a>(
         blocks: path_blocks,
         train_simulations: trains_details
             .iter()
-            .map(|detail| {
+            .enumerate()
+            .map(|(index, train_details)| {
                 (
-                    detail.train_id,
+                    index as i64,
```

Comment:
I see that the `usize` is converted to `i64` here, but converted back to `usize` on line 88 below. I'm guessing the ` i64` constraint come from the definition of `SignalUpdatesRequest`. Would it make sense to change the type in `SignalUpdatesRequest`? (might be for another PR)